Makefile.ia32

```diff
@@ -10,12 +10,15 @@ CROSS ?= i386-pc-phoenix-
 
 CC := $(CROSS)gcc
 
+DEFAULT_OLVL := -O2
 ifeq ($(DEBUG), 1)
-  CFLAGS += -Og
+  OLVL := -Og
 else
-  CFLAGS += -O2 -DNDEBUG
+  OLVL := $(or $(OLVL),$(DEFAULT_OLVL))
+  CFLAGS += -DNDEBUG
 endif
 
+CFLAGS += $(OLVL)
```

Comment:
The same code is for each target. It can move it to a `Makefile.common`.

Reply:
I'm not sure. I feel that some part of  this is same as a coincidence, not the rule:
- Probably we want to have different default optimisation level for each target architecture
- I don't ser any reason to not to define DNDEBUG and do not use -Og in debug builds. So ack for that.

I move DEBUG logic to Makefile.common. If any others will have any objection, please let me know.